sdk/python/feast/infra/materialization/contrib/bytewax/bytewax_materialization_dataflow.py

```diff
@@ -29,7 +28,16 @@ def __init__(
         self._run_dataflow()
 
     def process_path(self, path):
-        fs = s3fs.S3FileSystem()
+        if path.startswith("s3://"):
```

Comment:
`fsspec` does that automatically. 

So I think you could just remove `s3fs` and change `dataset = pq.ParquetDataset(path, use_legacy_dataset=False)`

Reply:
Great, thanks for the advice

Let me update the PR